frontend/src/assets/logo-white.svg

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
if we're going to have logos in SVG can we also store them in https://github.com/CodeForPoznan/Community/tree/master/logos